# file: C:\Users\dsuth\AppData\Roaming\Python\Python312\site-packages\pydantic\errors.py
# hypothesis_version: 6.137.1

[".*'(.+?)'", 'PydanticErrorCodes', 'PydanticImportError', 'PydanticUserError', 'config-both', 'custom-json-schema', 'discriminator-alias', 'import-error', 'removed-kwargs', 'root-model-extra', 'typed-dict-version', 'undefined-annotation', 'validator-no-fields', 'validator-signature']